.prettierrc.json

```diff
@@ -2,7 +2,7 @@
   "tabWidth": 2,
   "trailingComma": "all",
   "singleQuote": true,
-  "printWidth": 120,
+  "printWidth": 80,
```

Comment:
Not sure which width is preferable, but either way can we keep things separate for PRs please? Things changes some files that just distract form the actual improvements made.

Reply:
I will revert this change

Reply:
It's already done in the change below. Isn't it a bit too narrow anyway? But we can follow on this if you like, just need to `prettier` all the files.

Reply:
I think prettier should never have a bigger print width of 80 as this can produce overly complicated code as it tries to fit as much code as it can into one line.